app/views/shared/_navbar.html.erb

```diff
@@ -1,10 +1,3 @@
-<% if user_masquerade? %>
```

Comment:
I removed this because the user needs to define the routes for pretender as per the gem. 

Should we create a controller for using pretender?

Reply:
Yes, we need to add an impersonates controller. I figure we can put it under `admin/users/impersonates_controller`

Reply:
We can put it under `madmin`

The problem is you need to setup your database first. Madmin looks at the db to generate resources, so it needs to be already migrated. Some people need to tweak the database.yml to set it up, so it really needs to be like two separate steps.